libs/device/feature/src/lib/api-entities/api-entities.service.ts

```diff
@@ -201,4 +201,34 @@ export class APIEntitiesService {
     }
     return this.handleSuccess(data)
   }
+
+  @IpcEvent(APIEntitiesServiceEvents.EntityDataDelete)
+  public async deleteEntityData({
+    entitiesType,
+    entityId,
+    deviceId,
+  }: {
+    entitiesType: string
+    entityId: EntityId
+    deviceId?: DeviceId
+  }): Promise<ResultObject<undefined>> {
```

Comment:
You can use this as a type and apply it in`deleted-entity-data.request.ts` file.